pkg/ec2pricing/ec2pricing.go

```diff
@@ -48,7 +48,8 @@ type spotPricingEntry struct {
 // New creates an instance of instance-selector EC2Pricing
 func New(sess *session.Session) *EC2Pricing {
 	return &EC2Pricing{
-		PricingClient: pricing.New(sess),
+                // use us-east-1 since pricing only has endpoints in us-east-1 and ap-south-1
+		PricingClient: pricing.New(sess.Copy(aws.NewConfig().WithRegion("us-east-1"))),
```

Comment:
nit: move the comment and value to const block

Reply:
I assume you mean us-east-1 to const. I actually think it's more clear to leave it as a string literal here. I think the comment is clearest here as well since it's a bit weird to see PricingClient and the other client initialized with a different config and warrants explanation as close as possible